var/spack/repos/builtin/packages/py-merlin/package.py

```diff
@@ -23,14 +24,18 @@ class PyMerlin(PythonPackage):
     depends_on("python@3.6:", type=("build", "run"))
     depends_on("py-setuptools", type=("build", "run"))
     depends_on("py-cached-property", type=("build", "run"))
-    depends_on("py-celery@5.0.0+redis+sqlalchemy", when="@1.7.5:", type=("build", "run"))
+    depends_on("py-celery@5.0.3:+redis+sqlalchemy", when="@1.8.0:", type=("build", "run"))
+    depends_on("py-celery@4.4.5:+redis+sqlalchemy", when="@1.7.5:", type=("build", "run"))
     depends_on("py-celery@4.3.0:4+redis+sqlalchemy", when="@:1.7.4", type=("build", "run"))
     depends_on("py-coloredlogs@10.0:", type=("build", "run"))
```

Comment:
Where is this version restriction coming from?

Reply:
Dunno, it's there since the start https://github.com/spack/spack/pull/15453.